src/context/EarnManagerProvider/components/Deposit/Deposit.tsx

```diff
@@ -313,32 +316,33 @@ export const Deposit = ({
                     rules={fiatInputValidation}
                   />
                 )}
-                <InputRightElement>
-                  <Popover>
-                    <PopoverTrigger>
-                      <IconButton
-                        size='sm'
-                        aria-label='Slippage Settings'
-                        variant='ghost'
-                        icon={<SliderIcon />}
-                      />
-                    </PopoverTrigger>
-                    <PopoverContent width='sm'>
-                      <PopoverArrow />
-                      <PopoverCloseButton />
-                      <PopoverHeader>
-                        {/* TODO translate */}
-                        <Text fontSize='sm' translation='modals.deposit.slippageSettings' />
-                      </PopoverHeader>
-                      <PopoverBody>
-                        <Slippage
-                          onChange={handleSlippageChange}
-                          value={values?.slippage || '0.5'}
+                {enableSlippage && (
+                  <InputRightElement>
+                    <Popover>
+                      <PopoverTrigger>
+                        <IconButton
+                          size='sm'
+                          aria-label='Slippage Settings'
+                          variant='ghost'
+                          icon={<SliderIcon />}
                         />
-                      </PopoverBody>
-                    </PopoverContent>
-                  </Popover>
-                </InputRightElement>
+                      </PopoverTrigger>
+                      <PopoverContent width='sm'>
+                        <PopoverArrow />
+                        <PopoverCloseButton />
+                        <PopoverHeader>
+                          <Text fontSize='sm' translation='modals.deposit.slippageSettings' />
+                        </PopoverHeader>
+                        <PopoverBody>
+                          <Slippage
+                            onChange={handleSlippageChange}
+                            value={values?.slippage || '0.5'}
```

Comment:
pull this out to default slippage const because we'll eventually want to save this as a user pref somewhere